### PR TITLE
{bio}[foss/2016a] 3d-dna v1.0.0

### PR DESCRIPTION
**Pull request by timeu**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/0/3d-dna/3d-dna-1.0.0-foss-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+version = '1.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/theaidenlab/3d-dna'
+description = """3D de novo assembly (3D DNA) pipeline"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/theaidenlab/%(name)s/archive']
+sources = ['master.zip']
```
[the remark this answers is not in the document]
Comment (timeu) [reply easybuild/easyconfigs/0/3d-dna/3d-dna-1.0.0-foss-2016a-Python-2.7.11.eb]: Created an issue, let's see what they see. If they don't do any releases I will update the PR with commit hash

[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/0/3d-dna/3d-dna-1.0.0-foss-2016a-Python-2.7.11.eb]: @timeu Any updates?

If we want to pursue this, we should update to a more recent toolchain too...

[the remark this answers is not in the document]
Comment (timeu) [reply easybuild/easyconfigs/0/3d-dna/3d-dna-1.0.0-foss-2016a-Python-2.7.11.eb]: unfortuntely no response from the developers.
The last real code commit is from Sep. 2018 (https://github.com/theaidenlab/3d-dna/commits/master). 
I think our best bet is to use the commit hash and version it as 0.0.1 and update the toolchain.
I am also not sure, if there are no better libraries for the specific pipeline out there. I will ask one of our scientists who used it.

[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/0/3d-dna/3d-dna-1.0.0-foss-2016a-Python-2.7.11.eb]: https://github.com/theaidenlab/3d-dna#overview-of-the-pipeline mentions a software version, I guess we should use that?

Looks like `<year><month><day>`...

[the remark this answers is not in the document]
Comment (timeu) [reply easybuild/easyconfigs/0/3d-dna/3d-dna-1.0.0-foss-2016a-Python-2.7.11.eb]: Good catch. Will close this PR and create a new one. 
Which toolchain should I update it to ? 2018b (this one we have in production). We also have 2019a.

[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/0/3d-dna/3d-dna-1.0.0-foss-2016a-Python-2.7.11.eb]: Up to you, more recent is better if you have the choice I'd say...

**Comment by timeu** [comment]
Closing this in favor of https://github.com/easybuilders/easybuild-easyconfigs/pull/9258

**Pull request closed without merging**